src/client/api/json_parser/output_basic.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef __CLIENT_API_JSON_PARSER_OUTPUT_BASIC_H__
+#define __CLIENT_API_JSON_PARSER_OUTPUT_BASIC_H__
+
+#include "client/api/json_parser/json_utils.h"
+#include "core/models/outputs/output_basic.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Deserialize JSON data to output basic object
+ *
+ * @param[in] output_obj An output JSON object
+ * @param[out] basic A new basic object
+ * @return int 0 on success
+ */
+int json_output_basic_deserialize(cJSON* output_obj, output_basic_t** basic);
+
+/**
+ * @brief Serialize basic output to JSON object
+ *
+ * @param basic An basic output
+ * @return cJSON* NULL on errors
```

Comment:
```suggestion
 * @return cJSON* or NULL on errors
```

Reply:
`cJSON*` is not part of the description sentence.

Reply:
I was referencing to below cases :
- [@return output_basic_t* or NULL on failure](https://github.com/iotaledger/iota.c/pull/339/files#diff-78a43ec0f14f3466dc2ec2c190be26668412bc20d60d46e7224171f87e371465R37)
- [@return output_basic_t* or NULL on failure](https://github.com/iotaledger/iota.c/pull/339/files#diff-78a43ec0f14f3466dc2ec2c190be26668412bc20d60d46e7224171f87e371465R72)
- [@return tagged_data_payload_t* or NULL on failure](https://github.com/iotaledger/iota.c/pull/339/files#diff-df9b6d3d491a8931542871c181747e15f60c8f18515da2e97b8024d6595d668bR77)

Reply:
documentation is not included in this PR, if you are happy I can fix the above sentences, what is your suggestion?

Reply:
We can fix this while doing the documentation part.